components/org.wso2.carbon.identity.query.saml/src/main/java/org/wso2/carbon/identity/query/saml/SignKeyDataHolder.java

```diff
@@ -109,6 +110,9 @@ public SignKeyDataHolder(String tenantDomain) throws IdentitySAML2QueryException
             } else {
                 keyAlias = ServerConfiguration.getInstance().getFirstProperty(
                         SECURITY_KEY_STORE_KEY_ALIAS);
+                if (StringUtils.isBlank(keyAlias)) {
+                    throw new IdentityException("Invalid file configurations. The key alias is not found.");
```

Comment:
please change the message, have to pinpoint which config element is incorrect and what is the current value.
e.g. "Security KeyStore KeyAlias" configuration in the server config.